engine/src/main/java/org/hibernate/validator/cfg/context/AnnotationProcessingOptions.java

```diff
@@ -13,10 +13,13 @@
  * @author Hardy Ferentschik
  */
 public interface AnnotationProcessingOptions<C extends AnnotationProcessingOptions<C>> {
+
 	/**
 	 * Specifies that annotations specified on the configured type or property should be ignored.
 	 *
 	 * @return Returns itself for method chaining.
+	 * @deprecated Use {@link AnnotationIgnoreOptions#ignoreAnnotations(boolean)} instead.
 	 */
+	@Deprecated
```

Comment:
Because you cannot "turn off" the ignore-annotations setting inherited from elements up in the hierarchy as you could with XML. I created that new interface to leave the current hierarchy unaltered while not forcing all the newly supported elements (methods, parameters etc.) to implement the deprecated method.